tox.ini

```diff
@@ -1,9 +1,27 @@
 [tox]
 envlist =
-    py27, pypy, py34, py35, py36
+    flake8, py27, pypy, py34, py35, py36, py37, coverage
 
 [testenv]
+usedevelop = true
 commands =
     zope-testrunner --test-path=src []
+extras =
+    test
+    stemmer
+
+[testenv:coverage]
+basepython =
+    python3.6
+commands =
+    coverage run -m zope.testrunner --test-path=src []
+    coverage html
+    coverage report --fail-under=98.5
```

Comment:
Please swap the order of `coverage report` and `coverage html` so people can see the results while waiting the interminable milliseconds while the HTML report is being generated.

Also please add `-m` to `coverage report` so people can see where the missing lines are.

Reply:
(I take this back: while it's convenient to see the numbers as soon as they're available, without waiting a couple of extra seconds for `coverage html` to run (my motivation), it's bad when coverage report --fail-under fails and then you can't look at the HTML report because it's not available.)